ui: statement details

vladlos opened the pull request:
updated retries font color when counts are 0

Resolves: #45448

Release note (ui): none

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/45531)
<!-- Reviewable:end -->

vladlos committed:
ui: statement details

updated retries font color when counts are 0

Resolves: #45448

Release note (ui): none

dhartunian commented:
bors r+
